.github/workflows/cypress.yml

```
@@ -10,7 +10,14 @@ on:
 
 # This workflow is made up of one job that calls the reusable workflow in graasp-deploy
 jobs:
+
   graasp-deploy-cypress-workflow:
+
+    # only cancel in-progress jobs or runs for the current workflow
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

`github.head_ref` may be more appropriate for PRs: it ensures that only one run per branch can be performed (`github.ref` is fully formed, so there may be discrepancies between e.g. a branch run and a PR run, see https://docs.github.com/en/actions/learn-github-actions/contexts#github-context however we stil must provide `github.ref` as default, since `github.head_ref` does not exist outside of PRs)
```suggestion
      group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
```